CONTRIBUTING.md

```diff
@@ -1,6 +1,21 @@
 # How to Contribute
 
-We're excited you're interested in contributing to Marquez! We'd love your help, and there are plenty of ways to contribute:
+We're excited you're interested in contributing to Marquez! We'd love your help.
+
+We welcome all levels of expertise from novice to experts.
+
+If you are `very new` to the open source contribution:

```

Comment:
Yep, I think for those who are new to OSS, we can provide helpful links like [Using the Fork-and-Branch Git Workflow](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/) (also linked in the _Resource_ section)

I would still keep the original list though, but now have links on how to fork and submit a PR:

```markdown
* Fix or [report](https://github.com/MarquezProject/marquez/issues/new) a bug
* Fix or improve documentation
* Pick up a ["good first issue"](https://github.com/MarquezProject/marquez/labels/good%20first%20issue), then [fork](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/) repository and send a pull request our way
```

The above markdown would render as:

* Fix or [report](https://github.com/MarquezProject/marquez/issues/new) a bug
* Fix or improve documentation
* Pick up a ["good first issue"](https://github.com/MarquezProject/marquez/labels/good%20first%20issue), then [fork](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/) repository and send a [pull request](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#submitting-a-pull-request) our way